core/models/group.py

```diff
@@ -182,9 +182,10 @@ def approve_quota(self, request_id):
         super(IdentityMembership, self).save()
         try:
             from service.tasks.admin import set_provider_quota,\
```

Comment:
(opinion) Would love to standardize on the parenthesis syntax for line-breaking imports over the backslash syntax.  The backslashes seem so...brutish.